apicast/src/policy.lua

```diff
@@ -0,0 +1,26 @@
+local _M = {}
+
+local setmetatable = setmetatable
+local policy_chain = require('policy_chain')
```

Comment:
How about requiring only what is needed? `require('policy_chain').PHASES`

Reply:
That would not really save anything. The whole file is executed and evaluated anyway. 
And it would require unrolling it back when we need second property from that module.

Reply:
:+1:
`policy` no longer depends on `policy_chain`, so this no longer applies.